### PR TITLE
Improve lighthouse score

### PR DESCRIPTION
**Pull request by roachnt**
Added PWA plugins and lang attribute.

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -2,5 +2,20 @@ module.exports = {
...
+    {
+      resolve: `gatsby-plugin-manifest`,
+      options: {
+        name: 'gatsby-starter-default',
+        short_name: 'starter',
+        start_url: '/',
+        background_color: '#f7f0eb',
+        theme_color: '#a2466c',
+        display: 'minimal-ui',
+        // icon: 'src/images/icon.png', // This path is relative to the root of the site.
```
Comment (KyleAMathews) [inline gatsby-config.js]: You want to just add the Gatsby logo? https://github.com/gatsbyjs/gatsby/blob/master/www/src/assets/gatsby-icon.png

File: package.json [hunk 2]
```diff
@@ -6,6 +6,8 @@
   "dependencies": {
     "gatsby": "next",
     "gatsby-plugin-react-helmet": "next",
+    "gatsby-plugin-manifest": "^1.0.27",
```
Comment (KyleAMathews) [inline package.json]: There should be `next` versions of these too

**Comment by roachnt** [comment]
Oh yes I'll add the logo and I'll change those versions, thanks!

**Comment by roachnt** [comment]
I also changed both the theme and background colors to the purple in the logo.

**Pull request merged**